packages/mco/utils/common.ts

```diff
@@ -0,0 +1,11 @@
+import { utcDateTimeFormatterWithTimeZone } from '@odf/shared/details-page/datetime';
+import { groupVersionFor, referenceFor } from '@odf/shared/utils';
+import { ObjectReference } from '@openshift-console/dynamic-plugin-sdk';
+
+export const formatDateTime = (lastAvailableTime: string) =>
+  utcDateTimeFormatterWithTimeZone.format(new Date(lastAvailableTime));
+
+export const getGVK = (objectRef: ObjectReference) => {
```

Comment:
```suggestion
export const referenceForObjectRef = (objectRef: ObjectReference) => {
```
based on current naming we are using in "/shared/utils/common" ??